packages/agoric-cli/template/contract/myFirstDapp.js

```diff
@@ -1,120 +1,125 @@
 /* eslint-disable no-use-before-define */
 import harden from '@agoric/harden';
-import { makeHelpers } from '@agoric/zoe/src/contracts/helpers/userFlow';
-import { makeMint } from '@agoric/ertp';
+import { makeHelpers, defaultAcceptanceMsg } from './helpers/userFlow';
```

Comment:
I think this line should still reference the @agoric/zoe package

Reply:
done.